README.md

```diff
@@ -52,12 +52,20 @@ NeoBundle 'hrsh7th/vim-vsnip-integ'
 ```viml
 " You can use other key to expand snippet.
 imap <expr> <C-j>   vsnip#available(1)  ? '<Plug>(vsnip-expand)'         : '<C-j>'
+" Expand selected placeholder with <C-j>
+smap <expr> <C-j>   vsnip#expandable()  ? '<Plug>(vsnip-expand)'         : '<C-j>'
 imap <expr> <C-l>   vsnip#available(1)  ? '<Plug>(vsnip-expand-or-jump)' : '<C-l>'
+" Jump to the next placeholder with <C-l>
 smap <expr> <C-l>   vsnip#available(1)  ? '<Plug>(vsnip-expand-or-jump)' : '<C-l>'
 imap <expr> <Tab>   vsnip#available(1)  ? '<Plug>(vsnip-jump-next)'      : '<Tab>'
 smap <expr> <Tab>   vsnip#available(1)  ? '<Plug>(vsnip-jump-next)'      : '<Tab>'
 imap <expr> <S-Tab> vsnip#available(-1) ? '<Plug>(vsnip-jump-prev)'      : '<S-Tab>'
 smap <expr> <S-Tab> vsnip#available(-1) ? '<Plug>(vsnip-jump-prev)'      : '<S-Tab>'
+" Select text to use as $TM_SELECTED_TEXT in the next snippet.
+" See https://github.com/hrsh7th/vim-vsnip/pull/50
+nmap        <C-l>   <Plug>(vsnip-select-text)
+xmap        <C-l>   <Plug>(vsnip-select-text)
+smap        <C-l>   <Plug>(vsnip-select-text)
```

Comment:
Thank you.

Could you change a usage of mapping to `<Plug>(vsnip-select-text)c` instead?

I think `<Plug>(vsnip-select-text)c` is useful than current suggestion.

What do you think about it?

Reply:
I think we should suggest

```viml
nmap        <C-l>   <Plug>(vsnip-select-text)
xmap        <C-l>   <Plug>(vsnip-select-text)
smap        <C-l>   <Plug>(vsnip-select-text)
nmap        <C-j>   <Plug>(vsnip-select-text)c
xmap        <C-j>   <Plug>(vsnip-select-text)c
smap        <C-j>   <Plug>(vsnip-select-text)c
```

Reply:
I use this mapping in `xmap`.

In `smap` you can just type and don't need to remove the text first.

In `nmap` we would have to provide a special mapping to reuse the text object.
E.g. `nmap <Plug>(vsnip-select-text-c)`

So I would suggest this mapping for `xmap`.
The `nmap` currently is like `y`, if you want I could implement a `c` version as well.
I'm not sure a `d` version would be useful though. But `c` is basically `d{*}a`. (So it wouldn't need much to support it as well)

Reply:
If you want to support something like this, I'd suggest either:

`<Plug>(vsnip-select-text-y)`
`<Plug>(vsnip-select-text-c)`
`<Plug>(vsnip-select-text-d)`

Here it easy to say how they behave. But in some cases two or all would be totally equal.

Alternatively:

`<Plug>(vsnip-select-text)`
`<Plug>(vsnip-cut-text)`

Where `cut` would be like the one with `c` and always ends in insert-mode with the text removed.

Reply:
Your opinion seems correct. I think we should suggest `<Plug>(vsnip-select-text)c` for only `xmap`.

For other modes/mappings seems good not to suggest anything.

Reply:
Added `<Plug>(vsnip-cut-text`) 👍